syntax = "proto3";

package toyBasket;

option cc_generic_services = true;

message EchoRequest {
  string message = 1;
}

message EchoResponse {
  string message = 1;
}

service EchoServer {
  rpc Echo(EchoRequest) returns(EchoResponse);
  rpc Print(EchoRequest) returns(EchoResponse);
}

